tests/runtime/probe

```diff
@@ -278,6 +278,18 @@ REQUIRES bpftool
 REQUIRES_FEATURE uprobe_multi
 BEFORE ./testprogs/uprobe_test
 
+NAME kprobe_session_progs
+RUN {{BPFTRACE}} --unsafe -e 'kprobe:ksys_* { printf("progs: "); system("bpftool prog | grep kprobe | grep ksys_ | wc -l"); exit(); } kretprobe:ksys_* { exit(); }'
```

Comment:
We don't want to print something in the exit probe here and confirm it was actually printed?

Reply:
> it should have the same system call in the kretprobe part.

But we should still assert that both the entry/exit paths were called.

Reply:
So I ended up using one large tests for all - checking that both probes fire and that the number of progs/links is 1. We could theoretically split it into multiple cases but IMO it's not that important feature that it would need multiple test. And I plan to drop the `system()` calls in future so the test will be much simplified.

Reply:
Nice! LGTM - A super nit would be to put the `RUN` part into a separate script for easier readability but not a blocker.

Reply:
Let's just keep it here for now. If I manage to move the `system` calls into separate `EXPECT` checks, the remainder program will be very simple.